doc/vagrant/share/packelektra.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Yes, but for these two use cases it will be good to have a script that already does everything? (Maybe installing is not always necessary, we could add command-line arguments for the script)